internal/services/datadog/datadog_monitors_resource_test.go

```diff
@@ -0,0 +1,286 @@
+package datadog_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/datadog/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type DatadogMonitorResource struct{}
+
+func TestAccDatadogMonitor_basic(t *testing.T) {
```

Comment:
We can also skip these tests if those environment variables aren't set
``` 
if os.Getenv("ARM_TEST_DATADOG_API_KEY") == "" || os.Getenv("ARM_TEST_DATADOG_APPLICATION_KEY") == "" {
		t.Skip("Skipping as ARM_TEST_DATADOG_API_KEY and/or ARM_TEST_DATADOG_APPLICATION_KEY are not specified")
		return
	}

Reply:
Currently this is mandatory field.

Reply:
These are mandatory fields but they might not be set locally or in our nightly testing environment. We'll want to add this so we know if we skipped them because we didn't set the variable versus it erroring because we didn't pass the environment variables in

Reply:
Done